qubes/tools/qubesd_query.py

```diff
@@ -11,7 +11,7 @@
     asyncio.ensure_future
 except AttributeError:
     # pylint: disable=no-member
-    asyncio.ensure_future = asyncio.async
+    asyncio.ensure_future = asyncio.ensure_future
```

Comment:
This doesn't make sense - if `asyncio.ensure_future` is already there, the assignment isn't needed. The whole `try/except` block is for compatibility with python 3.4. Can be dropped.